t.rb

```diff
@@ -0,0 +1,2 @@
+# bad
+[2, 1, 3].sort.first
```

Comment:
Use `min` instead of `sort...first`.

```suggestion
[2, 1, 3].min
```